docs/CHANGELOG.md

```diff
@@ -50,6 +50,8 @@ ETH_GAS_LIMIT_KEEPER_JOB_TYPE # EVM.GasEstimator.LimitKeeperJobType
 
 - Simplified the Keepers job spec by removing the observation source from the required parameters.
 
+- `Arbitrum` chains are no longer restricted to only `FixedPrice` `GAS_ESTIMATOR_MODE`
```

Comment:
Do we also want to change the default?

Reply:
I think not until after the mainnet event happens. This way node ops can upgrade to this release before the event, and then just reconfigure minimal settings during the Nitro upgrade downtime period.